src/Browser/Avalonia.Browser/Interop/NavigationHelper.cs

```diff
@@ -16,7 +16,7 @@ public static Task<bool> OnBackRequested()
             .OnBackRequested() ?? false;
         return Task.FromResult(handled);
     }
-    
-    [JSImport("globalThis.open")]
-    public static partial JSObject? WindowOpen(string uri, string target);
+
+    [JSImport("NavigationHelper.openUri")]
```

Comment:
```suggestion
    [JSImport("NavigationHelper.openUri", AvaloniaModule.MainModuleName)]
```

This one should be necessary though.

Reply:
Resolved.